src/frequenz/sdk/microgrid/_data_pipeline.py

```diff
@@ -300,15 +300,42 @@ def ev_charger_pool(ev_charger_ids: set[int] | None = None) -> EVChargerPool:
     created and returned.
 
     Args:
-        ev_charger_ids: Optional set of IDs of EV Charger to be managed by the
-            EVChargerPool.
+        ev_charger_ids: Optional set of IDs of EV Chargers to be managed by the
+            EVChargerPool.  If not specified, all EV Chargers available in the
+            component graph are used.
 
     Returns:
         An EVChargerPool instance.
     """
     return _get().ev_charger_pool(ev_charger_ids)
 
 
+def battery_pool(battery_ids: abc.Set[int] | None = None) -> BatteryPool:
+    """Return the corresponding BatteryPool instance for the given ids.
+
+    If a BatteryPool instance for the given ids doesn't exist, a new one is
+    created and returned.
+
+    Args:
+        battery_ids: Optional set of IDs of batteries to be managed by the
+            BatteryPool.  If not specified, all batteries available in the
```

Comment:
Do we want to use double space after a `.`? Is that a policy I missed? Same above

Reply:
We don't have a policy for or against it.  But either way, markdown renderers are able to keep them consistent when generating docs.